.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -4,45 +4,39 @@ about: Request a new feature
 labels: feature-request, status/needs-triage
 ---
 
-<!-- NOTE:📕 If issue regarding developer guide, please create issue [here](https://github.com/awsdocs/aws-cdk-guide/issues) -->
+<!-- short description of the feature you are proposing: -->
+
+
+
+
+### Use Case
+
+<!-- why do you need this feature? -->
 
-## :rocket: Feature Request
 
-### General Information
-<!--
-Check the box below (with an X) if you are able and willing to propose an
-implementation for the requested feature. This does not imply a commitment from
-you to actually do it!
--->
-* [ ] :wave: I may be able to implement this feature request
-<!--
-Check the box below (with an X) if you think this feature might result in a
-breaking change (this requiring a major version bump when released). If unsure,
-please leave the box un-checked.
--->
-* [ ] :warning: This feature might incur a breaking change
 
-### Description
-<!--
-Describe the feature you would like added in as much detail as you can. Make
-sure you explain how it'll enhance the project, user-experience or feature set.
--->
 
 ### Proposed Solution
-<!--
-Whenever relevant, describe how you would like the feature to be implemented.
-Include any documentation that can help understand your idea in very concrete
-ways, such as code examples that leverage your feature, captures of design
-diagrams, ...
+
+<!-- prototype/workaround/sketch/reference: -->
```

Comment:
What does this mean?

Reply:
I'll improve the wording and remove the proposed solution header